code/src/Components/Album.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import "./Album.css"
+import { ReactComponent as Heart } from "./heart.svg"
+import { ReactComponent as Play } from "./play.svg"
+import { ReactComponent as Dots } from "./dots.svg"
+
+
+
+export const Album = (props) => (
+
+  <div className="albumInfo">
+    <a href={props.albumLink}>
+      <img className="albumCover" src={props.image} alt="album cover" />
+    </a>
+    <h1>{props.title}</h1>
+    <a href={props.artistLink}>
+      <h3>{props.artist}</h3>
+    </a>
```

Comment:
Here, to be able to show multiple artists (if there are multiple artists on the album) do a similar solution as you did in App.js to get all the albums. 
```
{props.artists.map(artist => (
 <a href={artist.external_urls.spotify}>
      <h3>{artist.name}</h3>
    </a>
))}
```